opendevin/sandbox/docker/ssh_box.py

```diff
@@ -169,7 +169,7 @@ def setup_user(self):
 
     def start_ssh_session(self):
         # start ssh session at the background
-        self.ssh = pxssh.pxssh(echo=False)
+        self.ssh = pxssh.pxssh()
```

Comment:
Why are we re-enabling echo? Seems better than `lstrip(cmd)`

Reply:
i mean that's how the previous logic works - the most recent version with `echo=True` seems to causing issues.  We could potentially change this back, but it will involve more testing to confirm.

Reply:
You mean `echo=False` causes issues? What sort of issues are you seeing?

Reply:
I mean the implementation in https://github.com/OpenDevin/OpenDevin/pull/1417 has some problem passing testcases, so this PR is essentially some sort of "revert". We probably need to look at other libraries if they can handle this better.